travis/publish.sh

```diff
@@ -72,7 +72,7 @@ check_release_tag() {
 }
 
 is_release_commit() {
-  project_version=$(./mvnw help:evaluate -N -Dexpression=project.version|grep -v '\[')
+  project_version=$(./mvnw help:evaluate -N -Dexpression=project.version|sed -n '/^[0-9]/p')
```

Comment:
this makes sure no extra output like download or PWD end up in the version. Notably, this could cause a release to not sync to maven central automatically